apps/src/templates/manageStudents/ManageStudentsTable.story.jsx

```diff
@@ -0,0 +1,248 @@
+import React from 'react';
+import ManageStudentsTable from './ManageStudentsTable';
+
+const passwordAccountData = [
+  {
+    id: 1,
+    name: 'Caley',
```

Comment:
I think these are vague enough, but Asher one time gave me feedback that we don't want personally identifiable info like this in the repo. 🤷🏼‍♀️